slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/dao/ValidatorsDao.java

```diff
@@ -22,6 +22,21 @@
 
 public class ValidatorsDao {
 
+  public Validator insertIfNotExist(final Handle handle, final Bytes validator) {
+    final List<Validator> result =
+        handle
+            .createUpdate("INSERT INTO validators (public_key) VALUES (?) ON CONFLICT DO NOTHING")
```

Comment:
This query will always increment the validator id. At some point will overflow the ids. Though the range is so large it might no matter?